validation/validator.js

```diff
@@ -0,0 +1,172 @@
+const Validation = require('jsonschema').Validator;
+
+
+function validatePlante(plante) {
+  if (!plante) {
+    throw new Error('No data received');
+  }
+
+  const validateForMe = new Validation();
+
+  const planteSchema = {
```

Comment:
I could be wrong, but I think it would me more readable to divide this huge schema in sub-schemas (see "split schemas" on the github read me).

Also the fact that this schema is declared inside the `validatePlante()` function means a new instance is created each time the function is called. Not a terrible performance issue, but not a good practice either.

Reply:
The schema has been split in order to improve readability and moved outside the function to limit the number of instances.